src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -924,6 +1027,53 @@ private synchronized void regenerateAITasks() {
 		editorReady = true;
 	}
 
+	private void updateBreedingConditions() {
+		if (breedable.isSelected()) {
+			hasAI.setSelected(true);
+			hasAI.setEnabled(false);
+			this.breedTriggerItems.setEnabled(true);
+			this.tameable.setEnabled(true);
+		} else {
+			hasAI.setEnabled(true);
+			this.breedTriggerItems.setEnabled(false);
+			this.tameable.setEnabled(false);
+		}
+	}
+
+	private void updateTradingConditions() {

```

Comment:
This method should be merged with enableOrDisableFields, otherwise, it is possible to re-enable fields by editing other fields than the ones affecting trading.

A review of all parameters that need to be disabled for this to work 100% is also is still needed.

If there will be too many, we may need to reconsider this PR as already suggested by Defeatomizer